src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -146,33 +149,54 @@ private void createTemplateFeedbackQuestions(String courseId, String feedbackSes
         return new ArrayList<FeedbackQuestionAttributes>();
     }
 
-    private FeedbackSessionAttributes extractFeedbackSessionData() {
+    public FeedbackSessionAttributes extractFeedbackSessionData() {
         //TODO assert parameters are not null then update test
         //TODO make this method stateless
-        
+      
         FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
         newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
         newSession.setFeedbackSessionName(Sanitizer.sanitizeTitle(
                 getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME)));
         
-        newSession.setCreatedTime(new Date());
+        // getting the class name from which it's being called 
+        String className = new Exception().getStackTrace()[1].getClassName();
+        
+        if (className == "InstructorFeedbackEditSaveAction") {
```

Comment:
== compares object identity, you probably meant to use the .equals method to compare 2 strings